app/service/planning_svc.py

```diff
@@ -59,7 +59,8 @@ def __init__(self, data_svc, utility_svc):
 
             variables = re.findall(r'#{(.*?)}', decoded_test, flags=re.DOTALL)
             if variables:
-                relevant_facts = await self._build_relevant_facts(variables, operation.get('facts', []))
+                private_facts = await self._explode_private_facts(operation['id'], agent['id'])
```

Comment:
let's rename "private_facts" to "agent_facts". These are a list of all the agent's facts, not just their private ones.